client/src/components/geneExpression/cellSetButtons.js

```diff
@@ -38,6 +38,7 @@ class CellSetButton extends React.Component {
           type="button"
           disabled={differential.diffExp}
           onClick={this.set.bind(this)}
+          data-testid={`cellset_button_${eitherCellSetOneOrTwo}`}
```

Comment:
is there a style guide for ids?   I see use of both dash and underscore (eg, foo-bar, foo_bar).

Reply:
You caught me. I had meant to clean that up and forgot. Do you have a preference?

Reply:
I don't.   I would prefer whatever is typical style.  Taking a quick look at eslint rules for react, I don't see anything jumping out.    So I'd probably default to dash, eg, foo-bar, unless it violates a typical styling standard.

Reply:
fixed, I went with "foo-bar" and I'll rely on @colinmegill for additional tweaks